src/lib/ppx_coda/versioned.ml

```diff
@@ -467,18 +471,15 @@ let generate_let_bindings_for_type_decl_str ~options ~path type_decls =
     generate_versioned_decls ~asserted generation_kind type_decl
   in
   let type_name = type_decl.ptype_name.txt in
-  let has_type_params = not (List.is_empty type_decl.ptype_params) in
   (* generate version number for Rpc response, but not for query, so we
      don't get an unused value
    *)
-  if
-    unnumbered || has_type_params
-    || (generation_kind = Rpc && String.equal type_name "query")
+  if unnumbered || (generation_kind = Rpc && String.equal type_name "query")
   then versioned_decls
   else
     generate_version_number_decl inner3_modules type_decl.ptype_loc
       generation_kind
-    :: versioned_decls
+    @ versioned_decls
```

Comment:
Can you explain why this changed? It's not obvious after a couple of reads.

Reply:
Before, there was just
```
let version = ...
```
in a structure item.

Now, there is:
```
let version = ...
let _ = version
```
in a structure, which is a list of structure items. So need append instead of cons.

Reply:
But you were probably asking why the change?